src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_system_capability_updated_notification.cc

```diff
@@ -80,13 +80,13 @@ void OnSystemCapabilityUpdatedNotification::Run() {
       break;
     }
     case mobile_apis::SystemCapabilityType::VIDEO_STREAMING:
-      if (hmi_capabilities_.video_streaming_capability()) {
-        msg_params[strings::system_capability]
-                  [strings::video_streaming_capability] =
-                      *hmi_capabilities_.video_streaming_capability();
-      } else {
-        return;
+      if ((*message_)[strings::msg_params].keyExists(strings::app_id)) {
+        (*message_)[strings::params][strings::connection_key] =
+            (*message_)[strings::msg_params][strings::app_id];
+        (*message_)[strings::msg_params].erase(strings::app_id);
```

Comment:
@IGapchuk I think that app_id parameter will be erased automatically according to the Mobile API

Reply:
@LitvinenkoIra In current implementation SDL doesn't erase parameter. You can find the same approach in the existing code base - https://github.com/LuxoftSDL/sdl_core/blob/develop/src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_bc_system_capability_updated_notification_from_hmi.cc#L87. But I removed that logic from this notification and added to the `OnBCSystemCapabilityUpdatedNotificationFromHMI` as the class method `RemoveAppIdFromNotification`